backend/tests/test_db_integration.py

```
@@ -0,0 +1,332 @@
+    async def test_execute_with_fetch_one(self):
+        """Test execute helper with fetch='one'."""
+        with patch("database.get_cursor") as mock_get_cursor:
+            mock_cursor = AsyncMock()
+            mock_cursor.execute = AsyncMock()
+            mock_cursor.fetchone = AsyncMock(return_value={"id": 1})
+            mock_cursor.__aenter__ = AsyncMock(return_value=mock_cursor)
+            mock_cursor.__aexit__ = AsyncMock()
+            mock_get_cursor.return_value = mock_cursor
+            
+            import database
+            result = await database.execute("SELECT 1", fetch="one")
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)